### ISSUE TITLE
Tripal WS Bad Request

### ISSUE THREAD

**Issue by almasaeed2010** [tripal/tripal#921]
### System information

* Tripal Version: 3.x
* Drupal Version: 7.6
* PostgreSQL Version: 9.6
* PHP Version: 7.1

### Issue description

Remote field keeps giving me an error indicating that the remote server has returned a `Bad Request` response. However, when I visit the URL using the browser directly, I get an appropriate JSON response. 

#### Steps to reproduce

1. Add a remote site (Tripal -> Data Storage -> Remote Tripal Sites)
    1. We are using `https://hardwoodgenomics.org` version `v0.1`
1. Add a new `Remote Field` in the Organism bundle
1. Configure it to point to `Organism?genus=[taxrank__genus]&species=[taxrank__species]`
1. Use `common_name` as the field to display
1. Give the field a new term and save
1. Go to Manage display for the Organism bundle and enable the field

#### Error messages and screenshots

<img width="1052" alt="Screen Shot 2019-04-02 at 10 45 46 AM" src="https://user-images.githubusercontent.com/1512664/55413212-e0d28a00-5536-11e9-9dc4-0f396dabd8d4.png">

Here is the query URL as shown in the screenshot: https://hardwoodgenomics.org/web-services/content/v0.1/OBI:0100026/26/common_name

As you can see, the URL returns the correct response.

**Comment by almasaeed2010** [comment]
This issue is extremely puzzling 🤔 

I ran the same code that gets data from the same URL in `drush php` and it works. However, when ran as part of the whole page, it fails with `400: Bad Request`.

**Comment by spficklin** [comment]
@almasaeed2010 I just tested this and It seems to be working for me in two different cases:

Case 1 I created an organism that you have on hardwoods:
![case1](https://user-images.githubusercontent.com/1719352/56775573-78b14580-677c-11e9-890a-3e2c18456067.png)

Case 2: An organism not on hardwoods.
![case2](https://user-images.githubusercontent.com/1719352/56775574-7a7b0900-677c-11e9-8a31-229a87459571.png)

And just to confirm that the same query is being used I printed a dpm of the query_str:
![case3](https://user-images.githubusercontent.com/1719352/56775730-4522eb00-677d-11e9-8777-7886c9cb6ac6.png)

So, in short. I can't replicate the problem. Are you still having this issue?

**Comment by almasaeed2010** [comment]
It still does not work for us but if it works for you it's likely a configuration issue on our part. Also, no one else reported the issue which probably is another indicator that it is only our sites. I'll close for now and update this thread if I find the issue on our side.

Thanks for looking into this @spficklin!
